### DISCUSSION TITLE
Provision to add a company in seperate tab. Also add selection based company while adding a product

### DISCUSSION THREAD

**Discussion by devexpart** [drkNsubuga/PharmaSpot#48] [Ideas]
Hi, on the main window, kindly add a new form to add companies. This option should also be added while adding a product and should not be left empty. 

![image](https://github.com/user-attachments/assets/bee4f4fb-0916-4701-a906-4ee189a52c78)
![image](https://github.com/user-attachments/assets/26efe50f-ff25-4def-bb08-f880296f184f)

**Comment by drkNsubuga** [comment]
Hello,
Can you please provide a use case for this feature?
